packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -97,6 +83,46 @@ module.exports = {
         .relative(paths.appSrc, info.absoluteResourcePath)
         .replace(/\\/g, '/'),
   },
+  optimization: {
+    minimizer: [
+      new UglifyJsPlugin({
+        uglifyOptions: {
+          ecma: 8,
+          compress: {
+            warnings: false,
+            // Disabled because of an issue with Uglify breaking seemingly valid code:
+            // https://github.com/facebook/create-react-app/issues/2376
+            // Pending further investigation:
+            // https://github.com/mishoo/UglifyJS2/issues/2011
+            comparisons: false,
+          },
+          mangle: {
+            safari10: true,
+          },
+          output: {
+            comments: false,
```

Comment:
This might be a legal issue, because this removes license comments too.

Reply:
This is the same uglify options as before though, I just moved it from uglify contructor to the new minimizer options.

Reply:
I just ran into this myself...  FYI, the comments options can take a regular expression or a function.  e.g.  `comments: /(license|copyright)/gi`

Reply:
Also check if the default works for you

Reply:
Yep, it sure did.